HW11/optional_task.py

```diff
@@ -0,0 +1,13 @@
+class MyCustomException(Exception):
+    def __init__(self, message="Custom exception is occurred"):
+        self.message = message
```

Comment:
nit: this line can be deleted because the message is set inside `Exception` constructor